apps/emqx_dashboard/src/emqx_dashboard.appup.src

```diff
@@ -1,7 +1,7 @@
 %% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  %% we should always restart dashboard to make sure api rules/swagger is updated
-  [{<<".*">>,[{restart_application, emqx_dashboard}]}],
```

Comment:
?

Reply:
erlfmt changed it.
should we remove all `*.appup.src` ?

Reply:
yes, let's exclude appup.src files.